Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
```

Comment:
Is it possible that something we do support in UWP and Windows might not be able to be supported in Linux or Mac, if so then that is the scenario for needing a "Will not/Cannot Support".

Links are easy, from what I saw it didn't screw up the formatting. I was thinking that if we had an issue label for each scenario then we might be able to include a link to an issue query that would only show issues related to that scenario.